package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "content",
   "version": "1.0.0",
-  "description": "![Content logo](demisto_content_logo.png)",
+  "description": "![Content logo](xsoar_content_logo.png)",
```

Seems like a bug that we have this here. We can change to an empty line.

```suggestion
  "description": "",
```